src/vs/workbench/contrib/debug/browser/breakpointWidget.ts

```diff
@@ -216,6 +216,13 @@ export class BreakpointWidget extends ZoneWidget implements IPrivateBreakpointWi
 		this.centerInputVertically();
 	}
 
+	protected override _onWidth(widthInPixel: number): void {
```

Comment:
Maybe just call `this._onWidth` so this isn't duplicating logic? And the other one subtracts `113`, I suppose that's for padding?

Reply:
Ok so what happened to the `- 113`? Also please call `_onLayout` from `_onWidth`, it feels odd to do it the other way to me.